zng/docs/spec.md

```diff
@@ -12,6 +12,32 @@
 > Also, we are contemplating reducing the number of primitive types, e.g.,
 > the number of variations in integer types.
 
+* [1. Introduction](#1-introduction)
+* [2. The ZNG data model](#2-the-zng-data-model)
+* [3. ZNG Binary Format (ZNG)](#3-zng-binary-format-zng)
+  + [3.1 Control Messages](#31-control-messages)
+    - [3.1.1 Typedefs](#311-typedefs)
+      - [3.1.1.1 Record Typedef](#3111-record-typedef)
+      - [3.1.1.2 Array Typedef](#3112-array-typedef)
+      - [3.1.1.3 Set Typedef](#3113-set-typedef)
+      - [3.1.1.4 Union Typedef](#3114-union-typedef)
+      - [3.1.1.5 Alias Typedef](#3115-alias-typedef)
+    - [3.1.2 End-of-stream Markers](#312-end-of-stream-markers)
```

Comment:
```suggestion
    - [3.1.2 End-Of-Stream Markers](#312-end-of-stream-markers)
```

Reply:
@nwt: According to https://www.grammarcheck.net/capitalization-in-titles-101/:

> Never capitalize prepositions and conjunctions of four or fewer letters

So I've committed a change to capitalize "Stream" but not "of".

Reply:
@philrz: Good catch!